var/spack/repos/builtin/packages/sleef/package.py

```diff
@@ -7,10 +7,29 @@
 
 
 class Sleef(CMakePackage):
-    """SIMD Library for Evaluating Elementary Functions,
-    vectorized libm and DFT."""
+    """SIMD Library for Evaluating Elementary Functions, vectorized libm and DFT."""
 
-    homepage = "http://sleef.org"
-    url      = "https://github.com/shibatch/sleef/archive/3.2.tar.gz"
+    homepage = "https://sleef.org"
+    git      = "https://github.com/shibatch/sleef.git"
 
-    version('3.2', sha256='3130c5966e204e6d6a3ace81e543d12b5b21f60897f1c185bfa587c1bd77bee2')
```

Comment:
I don't like removing stable releases and replacing them with random commits, but that's what PyTorch uses and mixing release numbers and dates doesn't work very well for sorting.

Reply:
Note that no packages other than PyTorch (in my PR) depend on sleef, so this isn't breaking anything.

Reply:
can't you do `3.2-<date>`?

Reply:
We could, that's not a bad idea. Should I go with that?

Reply:
I kind of like that better.

@rblake any opinions on this?